src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -192,10 +221,10 @@ public string EncodedHeader
                 // TODO - need to account for JWE
```

Comment:
>JWE [](http://example.com/codeflow?start=46&length=3)

what is the plan for JWE? chcking the `alg` claim in the header and proceeding from there?

Reply:
This comment was already present. Need to check with Brent if this can be addressed separately.

Reply:
The comment in the code is not in scope for the current PR.